Styles/component/_header.css

```
@@ -0,0 +1,6 @@
+header {
```

If you add this to it's own component, it should be a class.
```suggestion
.header {
```